pages/linux/apk.md

```diff
@@ -0,0 +1,27 @@
+# apk
+
+> Alpine Linux package management tool.
+
+- Update repository indexes from all remote repositories:
```

Comment:
Shouldn't this be after the `add` option ? What is the natural flow of actions here ? Does the user first do an update or add a package ?

Reply:
I believe one has to first update the indexes, as with `apt`.

Reply:
Yes, looking at their docs that is the correct flow. It updates the package indexes. :+1:

Reply:
great.